examples/gatsby/gatsby-spa/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gatsby-spa",
+  "version": "1.0.0",
+  "private": true,
+  "description": "gatsby-spa",
+  "author": "ryunsong-contentful",
+  "keywords": [
+    "gatsby"
+  ],
+  "scripts": {
+    "develop": "gatsby develop",
+    "start": "gatsby develop",
+    "build": "gatsby build",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@contentful/experiences-sdk-react": "^1.27.1",
+    "gatsby": "^5.14.1",
```

Comment:
we should have a dep on the 'contentful' lib here as well as its a peer dep of the sdk.